tests/includes/awscli.sh

```diff
@@ -0,0 +1,21 @@
+setup_awscli_credential() {
+	if ! which aws >/dev/null 2>&1; then
+		sudo snap install aws-cli --classic || true
+	fi
+
+	export AWS_DEFAULT_PROFILE=default
+	if [ -f "$HOME/.aws/credentials" ]; then
+		return
+	fi
+
+	mkdir -p "$HOME"/.aws
+	echo "[default]" >"$HOME/.aws/credentials"
```

Comment:
Uh, this makes the shell tests unsafe to run on your own machine.

Reply:
Relying on jenkins jobs to set stuff up and without a way to run the tests locally is not ideal. So we need to find a compromise approach. We could use a profile other than "default" and look for it in the local creds and add it if missing. I was looking for a quick win initially to unlock the tests.

Reply:
All good, I missed the check above.